README.md

```diff
@@ -6,11 +6,35 @@
 For updates and history of development please see [this page](https://github.com/amiaopensource/audiorecorder/blob/master/history.md).
 
 ## Installation:
-Can be installed on macOS via Homebrew with the following command: 
+
+audiorecorder can be installed via [homebrew](https://brew.sh/) on macOS and [linuxbrew](http://linuxbrew.sh/) on Linux.
+
+__MacOS Installation__:
+Audiorecorder can be installed via Homebrew with the following command:
 
 `brew tap amiaopensource/amiaos && brew install audiorecorder`.
```

Comment:
As it is a command, I would suggest to delete the ending `.` because it males it easier to copy&paste for beginners.

I am always hesitating if two commands (combined in one or not)
```
brew tap amiaopensource/amiaos
brew install audiorecorder
```
or a single command
```
brew install amiaopensource/amiaos/audiorecorder
```
is the better idea… I really don’t know.